src/main/java/de/hysky/skyblocker/config/configs/UIAndVisualsConfig.java

```diff
@@ -128,9 +129,17 @@ public static class TabHudConf {
         @SerialEntry
         public boolean enableHudBackground = true;
 
+        @SerialEntry
+        public boolean effectsFromFooter = false;
+
+        @Deprecated
         @SerialEntry
         public boolean plainPlayerNames = false;
```

Comment:
This option should be deleted

Reply:
Deleting is fine, but we should discuss this. Imo these should just be deprecated to prevent possibilities of conflicts.